src/dynodeController.ts

```diff
@@ -43,6 +42,7 @@ class NodeController {
     this.model.addEventListener('inputsReady', inputsReady);
     this.model.addEventListener('afterResolve', afterResolve);
     this.model.addEventListener('nodeRemoved', nodeRemoved); // Perhaps this event belongs to the Network model?
+    // TODO Handle "nothing" in inputChange
     this.view.addEventListener('inputChange', (ev: VEvent) => {
       for (const [key, value] of Object.entries(ev.detail as Record<string, unknown>)) {
         node.getInput(key).setValue(value);
```

Comment:
This is critical: when user changes the input value in UI, for the model side that means the input **defaultValue** has to be set (instead of input value, through `setValue` method).